budcamp/Homework1/Homework1/Program.cs

```diff
@@ -0,0 +1,281 @@
+            double number1 = int.Parse(Console.ReadLine());
+            Console.WriteLine("enter second int");
+            double number2 = int.Parse(Console.ReadLine());
+            Console.WriteLine("enter third int");
+            double number3 = int.Parse(Console.ReadLine());
+
+            Console.WriteLine($"Largest value = {Math.Max(number1, Math.Max(number2, number3))}");
+            Console.WriteLine($"Lowest value = {Math.Min(number1, Math.Min(number2, number3))}");
+        }
+
+        private static void Task6()
+        {
+            Console.WriteLine("plz input your string");
+            string inputString = Console.ReadLine();
+            Console.WriteLine($"Result of the ex: {StrToUpperCase(inputString)}");
+        }
+
+        public static string StrToUpperCase(string input)
+        {
+            int length = input.Length;
+            int condition = 5;
+
+            if (length == condition)
+            {
+                return input.ToUpper();
+            }
+            else if (length > condition)
+            {
+                return input.Substring(0, 4).ToUpper() + input.Substring(5, length - 5);
+            }
+
+            return input;
+        }
+
+        private static void Task7()
+        {
+            Console.WriteLine("enter your int to check if it is multiple of 3 or 7");
+            int newInput = int.Parse(Console.ReadLine());
+            Console.WriteLine(IsMultipleOf3Or7(newInput));
+        }
+        public static bool IsMultipleOf3Or7(int input)
+        {
+            return input % 3 == 0 || input % 7 == 0;
+        }
+
+        private static void Task8()
+        {
+            Console.WriteLine("enter your string to check if it starts with 'C#'");
+            string str = Console.ReadLine();
+            Console.WriteLine($"result: {CheckStartOfString(str)}");
+        }
+
+        public static bool CheckStartOfString(string input)
+        {
+            string condition = "C#";
+
+            return input.StartsWith(condition);
+        }
+
+        private static void Task9()
+        {
+            Console.WriteLine("enter the circle radius");
+            int radius = int.Parse(Console.ReadLine());
+            Console.WriteLine($"Perimeter = {2 * radius * Math.PI} \nArea = {2 * Math.PI * radius * radius}");
+        }
+
+        private static void Task10()
+        {
+            Console.WriteLine(SumAndAverageOf10Num());
+        }
+
+        public static string SumAndAverageOf10Num()
+        {
+            double sum = 0;
+            double average;
+
+            Console.WriteLine("Input your numbers");
+
+            for (int i = 0; i < 10; i++)
+            {
+                double num = Double.Parse(Console.ReadLine());
+                sum += num;
+            }
+
+            average = sum / 10;
+
+            return $"Sum = {sum}\nAverage = {average}";
+        }
+
+        private static void Task11()
+        {
+            MultiplicationTable();
```

Comment:
You just can remove MultiplicationTable altogether and put it's content here instead